opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -394,12 +393,15 @@ message SummaryValue {
 
   // The total number of recorded values since start_time. Optional since
   // some systems don't expose this.
-  google.protobuf.Int64Value count = 3;
+  uint64 count = 3;
 
   // The total sum of recorded values since start_time. Optional since some
   // systems don't expose this. If count is zero then this field must be zero.
-  // This field must be unset if the sum is not available.
-  google.protobuf.DoubleValue sum = 4;
+  double sum = 4;
+
+  // count_and_sum_unspecified must be set to true if count and sum values are unknown or
+  // unspecified.
+  bool count_and_sum_unspecified = 5;
```

Comment:
I remember I saw cases when one was define but not the other.

Reply:
OK, I'll separate the fields.

Reply:
Fixed.